### ISSUE TITLE
useful: "query hda-info <hda_id>"

### ISSUE THREAD

**Issue by bgruening** [usegalaxy-eu/gxadmin#48]
Which information about the history, which dataset, and more importantly which job_id.

**Comment by hexylena** [comment]
yeah, I've considered writing a `report hda` for a while. nice idea, would be good to have!
